tests/queries/0_stateless/00727_concat.sql

```diff
@@ -1 +1,74 @@
-SELECT CONCAT('Hello', ', ', 'world!');
+-- Tags: no-fasttest
+-- no-fasttest: json type needs rapidjson library, geo types need s2 geometry
+
+-- not tested here: (Simple)AggregateFunction, Nested
+
```

Comment:
Would be great to include a few negative tests, e.g. calling `concat` with 0 arguments or with 1 arguments (these should throw errors).